go/tools/gazelle/resolve/resolve.go

```diff
@@ -77,6 +77,12 @@ func NewLabelResolver(c *config.Config) LabelResolver {
 	switch c.DepMode {
 	case config.ExternalMode:
 		e = newExternalResolver()
+	case config.KnownMode:
+		er := newExternalResolver()
```

Comment:
It would be better to pass additional imports to `newExternalResolver` instead of modifying the cache directly here.